app/uk/gov/hmrc/individualsincomeapi/controllers/SaIncomeController.scala

```diff
@@ -106,7 +106,18 @@ abstract class SaIncomeController(saIncomeService: SaIncomeService) extends Comm
     }
   }
 
-  def employmentsIncome(matchId: UUID, taxYearInterval: TaxYearInterval) = Action.async { implicit request =>
+  def saUkPropertiesIncome(matchId: UUID, taxYearInterval: TaxYearInterval) = Action.async { implicit request =>
+    requiresPrivilegedAuthentication("read:individuals-income-sa-uk-properties") {
+      saIncomeService.fetchSaUkPropertiesIncomeByMatchId(matchId, taxYearInterval) map { saForeignIncomes =>
```

Comment:
saUkPropertiesIncomes

Reply:
happy to change this, but we're using this form for all others, e.g. saTrustsIncome, saForeignIncome, saPartnershipsIncome, etc.